google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationPanel.java

```diff
@@ -52,6 +54,18 @@
   private HyperlinkLabel appEngineCostWarningLabel;
   private JLabel serviceLabel;
   private JCheckBox hiddenValidationTrigger;
+  private JTextField compileEncodingTextField;
```

Comment:
is it not possible to refactor this so that this UI fields live in `AppEngineStandardDeploymentEditor` instead of in this common panel?

Reply:
you could have the tabbed panel field live in the common panel, but then all of the inner components of the "advanced" tab live in `AppEngineStandardDeploymentEditor`